provider/resource_keycloak_openid_client_authorization_aggregate_policy.go

```diff
@@ -0,0 +1,185 @@
+package provider
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func resourceKeycloakOpenidClientAuthorizationAggregatePolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakOpenidClientAuthorizationAggregatePolicyCreate,
+		Read:   resourceKeycloakOpenidClientAuthorizationAggregatePolicyRead,
+		Delete: resourceKeycloakOpenidClientAuthorizationAggregatePolicyDelete,
+		Update: resourceKeycloakOpenidClientAuthorizationAggregatePolicyUpdate,
+		Importer: &schema.ResourceImporter{
+			State: resourceKeycloakOpenidClientAuthorizationAggregatePolicyImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"resource_server_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"realm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"decision_strategy": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"owner": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"logic": {
```

Comment:
Looks like this can only be `POSITIVE` or `NEGATIVE`, could you add some validation here to enforce this?  I think you can do this for all of the policy resources as well.